core/src/manifest/network.rs

```diff
@@ -24,4 +27,101 @@ pub struct Network {
 
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub disable_logs_bloom_checks: Option<bool>,
+
+    /// Reth configuration for this network
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub reth: Option<RethConfig>,
+}
+
+/// Configuration for Reth node and ExEx
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct RethConfig {
+    /// Whether to enable Reth integration
+    #[serde(default)]
+    pub enabled: bool,
+
+    /// CLI configuration for the Reth node
+    #[serde(default)]
+    pub cli_config: RethCliConfig,
+}
+
+/// CLI configuration for the Reth node
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct RethCliConfig {
```

Comment:
are all these fields optional in the reth solution - IMHO being strong for them to define it over putting fake values in the yaml would be more sense